cmatrix.c

```diff
@@ -708,6 +724,34 @@ if (console) {
                 }
             }
         }
+
+        //Check if computer is locked
+        if(lock == 1){
+
+            //Add our message to the screen
+            char *msg = "Computer locked.";
+            int msg_x = LINES/2;
+            int msg_y = COLS/2 - strlen(msg)/2;
+
+            //Add space before message
+            move(msg_x-1, msg_y-2);
+            for(int i = 0; i < strlen(msg)+4; i++)
+                addch(' ');
+
+            //Write message
+            move(msg_x, msg_y-2);
+            addch(' ');
+            addch(' ');
+            addstr(msg);
+            addch(' ');
+            addch(' ');
+
+            //Add space after message
+            move(msg_x+1, msg_y-2);
+            for(int i = 0; i < strlen(msg)+4; i++)
```

Comment:
Use c99 style for declaration here pls.

Reply:
I think I solved the issue. If not, do not hesitate to tell me...